pom.xml

```diff
@@ -193,7 +193,7 @@
         <dependency>
             <groupId>net.coreprotect</groupId>
             <artifactId>coreprotect</artifactId>
-            <version>19.3</version>
+            <version>19.5</version>
```

Comment:
Isn't the API much more up to date than this?

Reply:
It is, but we are only running 19.5 on the server. Didn't want to risk accidental breaking changes between API versions when we're not running 20.X on the live server yet.